.circleci/unittest/linux/scripts/environment.yml

```diff
@@ -12,7 +12,6 @@ dependencies:
   - ffmpeg=4.2
   - ca-certificates
   - pip:
-    - future
-    - pillow>=4.1.1
-    - scipy
-    - av
\ No newline at end of file
+      - future
+      - pillow>=4.1.1
+      - scipy
```

Comment:
Can you revert those changes (and the other ones from my testing branch)? They might interfere with your PR